commands/Music/clearqueue.js

```diff
@@ -0,0 +1,32 @@
+const { Command } = require("klasa");
+
+module.exports = class extends Command {
+
+    constructor(...args) {
+        super(...args, {
+            runIn: ["text"],
+            cooldown: 10,
+            aliases: ["cq", "prunequeue"],
+            permissionLevel: 0,
+            requiredPermissions: ["USE_EXTERNAL_EMOJIS"],
+            description: (msg) => msg.language.get("COMMAND_CLEARQUEUE_DESCRIPTION"),
+            extendedHelp: "No extended help available."
+        });
+        this.Music = true;
+    }
+
+    async run(msg) {
+        const music = msg.guild.music();
```

Comment:
music isnt a function and shouldnt be

Reply:
Well, that's the thing, Quantum made it this way so I've been using it like this ever since.

Reply:
its a horrible idea and should not be done like this